src/elements/content-sidebar/activity-feed/annotation-thread/useAnnotationThread.js

```diff
@@ -97,12 +97,16 @@ const useAnnotationThread = ({
         }
     };
 
-    useAnnotatorEvents({
+    const events = useAnnotatorEvents({
         onAnnotationDeleteStart: setAnnotationPending,
         onAnnotationUpdateEnd,
         onAnnotationUpdateStart: setAnnotationPending,
     });
 
+    React.useEffect(() => {
+        events.emitAnnotationActiveChangeEvent(annotationId, fileId);
+    }, [annotationId, events, fileId]);
```

Comment:
Is this stable? I think `events` has a lot in it and could end up changing somewhat often?

Reply:
I think you're right. We don't need events there. Maybe just `annotationId` and `fileId`. We want to emit this event only once when Annotation is displayed